src/client/camera.cpp

```diff
@@ -602,6 +602,32 @@ void Camera::drawWieldedTool(irr::core::matrix4* translation)
 	m_wieldmgr->drawAll();
 }
 
+void Camera::toggleCameraMode()
+{
+	switch(m_camera_mode) {
+		case CAMERA_MODE_FIRST: {
+			if (!m_camera_mode_restrictions.third)
+				m_camera_mode = CAMERA_MODE_THIRD;
+			else if (!m_camera_mode_restrictions.third_front)
+				m_camera_mode = CAMERA_MODE_THIRD_FRONT;
+			break;
+		}
+		case CAMERA_MODE_THIRD: {
+			if (!m_camera_mode_restrictions.third_front)
+				m_camera_mode = CAMERA_MODE_THIRD_FRONT;
+			else if (!m_camera_mode_restrictions.first)
+				m_camera_mode = CAMERA_MODE_FIRST;
+			break;
+		}
+		case CAMERA_MODE_THIRD_FRONT: {
+			if (!m_camera_mode_restrictions.first)
+				m_camera_mode = CAMERA_MODE_FIRST;
+			else if (!m_camera_mode_restrictions.third)
+				m_camera_mode = CAMERA_MODE_THIRD;
+		}
```

Comment:
A break is missing here. (https://dev.minetest.net/Code_style_guidelines#Miscellaneous)

Also there are many curly brackets. I don't know whether it looks weird to make one block per case.

And everything is indented too much (see https://www.kernel.org/doc/html/latest/process/coding-style.html#indentation).

Reply:
Thanks for pointing that out. Fixed. :+1: